### ISSUE TITLE
App cleaner deleted my spotify local files by default

### ISSUE THREAD

**Issue by NMBlanco** [d4rken-org/sdmaid-se#770]
So I dont know if its on purpose or not but today I did an app clean (App Reiniger) with the default settings (because with SD Maid 1 and its default settings I never had any issues with it) and it apparently deleted all the local files on one of my spotify playlists.
I have multiple playlists on my spotify where I sync local files from my computer. The local files from one of those playlists are all gone now, without any trace. 
This never happened before with SD Maid 1 and the same setup.

It's incredibly frustrating since I have to manually move all my local files to the phone again, which is just about 800 songs. 
My setup:
Motorola Edge 30 Ultra with Android 13, Stock Rom

**Comment by d4rken** [comment]
Which SD Maid update?

Do you have the offline files filter enabled?

If NOT, then please share a debug log of the scan or at least some screenshots of the files paths that should not be deleted.

**Comment by NMBlanco** [comment]
Using the latest version, 0.15.3.
I didnt have offline files filter enabled because I know it would mess with important files like spotify. And I just assumed it works (kind of) the same way like in the old SD Maid.
I think I cant get a log anymore, because it happened yesterday and I only noticed it a few hours later when I opened Spotify.

A picture of the files that shouldnt be deleted:
https://picr.eu/i/I2jKS
